.github/workflows/test.yaml

```diff
@@ -7,13 +7,3 @@ jobs:
   unit-tests:
     uses: canonical/operator-workflows/.github/workflows/test.yaml@main
     secrets: inherit
-  plugins-test:
-    name: Specific test for the plugin
-    runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v3
-      - name: Install tox
-        run: |
-          python -m pip install --upgrade pip
-          pip install tox
-      - run: tox -e plugins
```

Comment:
Could you please clean up `tox.ini` and remove the `plugins` env? I think it was copied from indico-operator and doesn't apply to this repository.